app/routes/_layout+/bookings.new.tsx

```diff
@@ -32,6 +32,8 @@ import { requirePermission } from "~/utils/roles.server";
  * This way all actions are available and its way easier to manage so in a way this works kind of like a resource route.
  */
 export async function loader({ context, request }: LoaderFunctionArgs) {
+  const url = new URL(request.url);
```

Comment:
Please use the way we already have for getting params on the server: `getCurrentSearchParams`

Reply:
Changed the way of accessing the query params. Resolved!